example/factories/__init__.py

```diff
@@ -58,3 +58,11 @@ class Meta:
     body = factory.LazyAttribute(lambda x: faker.text())
     author = factory.SubFactory(AuthorFactory)
 
+
+class EntryTaggedItemFactory(factory.django.DjangoModelFactory):
```

Comment:
Could you call this `TaggedItemFactory` instead of `EntryTaggedItemFactory`? That seems more in line with the other factory naming conventions here.